src/strings/PowerFxResources.en-US.resx

```diff
@@ -4051,6 +4051,10 @@
     <value>In namespace {1}, function {0} is deprecated.</value>
     <comment>A warning message when a function is deprecated.</comment>
   </data>
+  <data name="WarnFunctionUseBinary" xml:space="preserve">
+    <value>This function uses 'binary' format and might not work as expected.</value>
+    <comment>A warning message when a function is using binary type.</comment>
+  </data>  
```

Comment:
We'd like to have in connector resource, but mechanics aren't working. 

We did something similar for DV, but for some reason, not working here:
[https://github.com/microsoft/Power-Fx-Dataverse/pull/247](https://github.com/microsoft/Power-Fx-Dataverse/pull/247)

Reply:
done